frontend/main.js

```diff
@@ -183,6 +183,9 @@ async function startApp () {
       this.ephemeral.syncs = this.ephemeral.syncs.filter(id => id !== contractID)
     }
   }
+  sbp('okTurtles.events/on', NEW_CURRENT_SYNCS, (currentSyncs) => {
+    this.ephemeral.syncs = currentSyncs
+  })
```

Comment:
This seems like a handler that is supposed to update the frontend's understand of whatever contracts are currently being sync'd, but it doesn't seem like it's doing that because the only time the `NEW_CURRENT_SYNCS` event seems to be emitted is once at page load.

Reply:
`NEW_CURRENT_SYNCS` is emitted on a SW update.

Reply:
I see that, but shouldn't it be emitted whenever there are "new syncs"?

Reply:
Can you rename `NEW_CURRENT_SYNCS` to `CONTRACT_SYNCS_RESET`?